ParisBNF/abb/BNFabb93.xml

```diff
@@ -0,0 +1,265 @@
+                                            <locus from="81r" to="88v"/>
+                                        </item>
+                                        <item xml:id="q12" n="12">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="89r" to="96v"/>
+                                        </item>
+                                        <item xml:id="q13" n="13">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="97r" to="104v"/>
+                                        </item>
+                                        <item xml:id="q14" n="14">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="105r" to="112v"/>
+                                        </item>
+                                        <item xml:id="q15" n="15">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="113r" to="120v"/>
+                                        </item>
+                                        <item xml:id="q16" n="16">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="121r" to="128v"/>
+                                        </item>
+                                        <item xml:id="q17" n="17">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="129r" to="136v"/>
+                                        </item>
+                                        <item xml:id="q18" n="18">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="137r" to="144v"/>
+                                        </item>
+                                        <item xml:id="q19" n="19">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="145r" to="152v"/>
+                                        </item>
+                                        <item xml:id="q20" n="20">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="153r" to="160v"/>
+                                        </item>
+                                        <item xml:id="q21" n="21">
+                                            <dim unit="leaf">6</dim>
+                                            <locus from="161r" to="166v"/>
+                                        </item>
+                                        <item xml:id="q22" n="22">
+                                            <dim unit="leaf">5</dim>
+                                            <locus from="167r" to="172v"/>1 stub after 168 </item>
```

Comment:
stub after 2 (you must use the relative position in the quire)

Reply:
Do I understand correctly: I do not indicate the actual folium but the folium within the quire?

Reply:
yes